pallets/aggregate/src/lib.rs

```diff
@@ -485,6 +591,8 @@ pub mod pallet {
         Aggregation,
         /// The funds are held as storage deposit for a domain registration.
         Domain,
+        /// The funds are held as storage deposit for dispatch fees for a given domain.
+        DomainDispatchFee,
```

Comment:
Maybe I'm wrong, but why we need an additional `HoldReason` ? Can't we group everything into the `Domain` one ?

Reply:
I can do that yes but then let's change the comment on Domain currently as 

`/// The funds are held as storage deposit for a domain registration.`

In this case it will be something like `The funds are held as storage deposit for anything related to a Domain`